packages/math-input/src/components/keypad/index.tsx

```diff
@@ -51,44 +44,76 @@ function allPages(props: Props): ReadonlyArray<TabbarItemType> {
         pages.push("Geometry");
     }
 
+    if (props.extraKeys?.length) {
+        pages.push("Extras");
+    }
+
     return pages;
 }
 
-export default class Keypad extends React.Component<Props, State> {
-    state: State = {
-        selectedPage: "Numbers",
-    };
-
-    static defaultProps: DefaultProps = {
-        extraKeys: [],
-    };
-
-    render(): React.ReactNode {
-        const {selectedPage} = this.state;
-
-        const availablePages = allPages(this.props);
-
-        return (
-            <View>
-                <Tabbar
-                    items={availablePages}
-                    selectedItem={selectedPage}
-                    onSelectItem={(tabbarItem: TabbarItemType) => {
-                        this.setState({selectedPage: tabbarItem});
-                    }}
-                    style={styles.tabbar}
-                />
-                {selectedPage === "Numbers" && <NumbersPage {...this.props} />}
-                {selectedPage === "Extras" && <ExtrasPage {...this.props} />}
+export default function Keypad(props: Props) {
+    const [selectedPage, setSelectedPage] = useState<TabbarItemType>("Numbers");
+
+    const availablePages = allPages(props);
+
+    const {
+        onClickKey,
+        extraKeys = [],
+        multiplicationDot,
+        divisionKey,
+        preAlgebra,
+        logarithms,
+        basicRelations,
+        advancedRelations,
+    } = props;
+
+    return (
+        <View>
+            <Tabbar
+                items={availablePages}
+                selectedItem={selectedPage}
+                onSelectItem={(tabbarItem: TabbarItemType) => {
+                    setSelectedPage(tabbarItem);
+                }}
+                style={styles.tabbar}
+            />
+
+            <View
+                style={{
+                    display: "grid",
+                    gridTemplateColumns: "repeat(6, 1fr)",
+                    gridTemplateRows: "repeat(4, 1fr)",
+                    backgroundColor: "#DBDCDD",
+                    maxHeight: 200,
+                    maxWidth: 300,
+                }}
```

Comment:
Is there any value in moving these to the `styles` object?  One thing I can think of is to trim down some of the style noise here , but then I consider that maybe it's useful to see the actual styles here inline instead of having the indirection of putting them at the bottom of the file.